<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="groupBox"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameLine">
       <property name="placeholderText">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstNameLine">
       <property name="placeholderText">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="patronymicLine">
       <property name="placeholderText">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="studentIdCardLine">
       <property name="placeholderText">
        <string>Номер студенческого билета</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addStudentButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
